Tests/LibDNS/TestDNSResolver.cpp

```diff
@@ -93,8 +79,7 @@ TEST_CASE(test_tls)
         [&] -> ErrorOr<DNS::Resolver::SocketResult> {
             Core::SocketAddress addr = { IPv4Address::from_string("1.1.1.1"sv).value(), static_cast<u16>(853) };
 
-            TLS::Options options;
-            options.root_certificates_path = locate_ca_certs_file();
+            TLS::Options options = {};
```

Comment:
Since you're dropping this already, could you also remove the cert path from https://github.com/LadybirdBrowser/ladybird/blob/809e465e05352f3360f26ac3ab94e5bc19dbcfa6/Utilities/dns.cpp#L81 ?

Reply:
I suppose that could come in handy, so sure, let's do that.